domains/shahid.json

```
@@ -1,9 +1,26 @@
 {
-    "owner": {
+  "description": "i want to link my subdomain into zoho mail for email service, Please note that my subdomain(shahid.is-a.dev) is not for 'shahidadev.github.io' my domain is for 'shahidadev.github.io/shahid' .",
+  "owner": {
         "username": "shahidadev",
         "email": "mdgffkiller8@gmail.com"
-    },
+   },
     "records": {
-        "CNAME": "shahidadev.github.io"
-    }
+        "CNAME": "shahidadev.github.io" ,
+        "TXT": ["zoho-verification=zb84657564.zmverify.zoho.in", "v=spf1 include:zohomail.in ~all"],
+     "MX": [
+      {
+        "target": "mx.zoho.in",
+        "priority": 10
+      },
+      {
+        "target": "mx2.zoho.in",
+        "priority": "20"
```

priority should be an integer/number
```suggestion
        "priority": 20
```